src/store/cache/user/hooks.ts

```diff
@@ -35,20 +35,22 @@ import getActiveStake from 'utils/activeStake'
 type UseUsersProp = {
   sortBy?: SortUser
   limit?: number
+  filter?: 'isOperator' | 'isDelegator'
```

Comment:
Ha, I'm introducing a type for this in my timeline work too. Making a note to consolidate later on